tests/test/paywall.test.js

```diff
@@ -43,15 +47,14 @@ describe.skip('The Unlock Paywall', () => {
     await page.evaluate(() => {
       window.scrollBy(0, 200)
     })
-    await page.waitForFunction(() => window.frames.length)
+    await wait.forIframe()
     const paywallIframe = page.mainFrame().childFrames()[0]
-    await paywallIframe.waitForSelector('#Paywall_Headline')
+    await paywallIframe.waitForSelector('.headline')
     const headlineLocation = await paywallIframe.evaluate(
-      () =>
-        document.querySelector('#Paywall_Headline').getBoundingClientRect().top
+      () => document.querySelector('.headline').getBoundingClientRect().top
     )
     // paywall has grown to hide the content.
-    // actual value of the headline is 45.390625
+    // actual value of the headline is 438.390625
```

Comment:
why change this? if the comment is correct, then the test assertion should fail, because it is asserting that the headline is < 50, but 438 is > 50